CrowdedMod/Patches/GenericPatches.cs

```diff
@@ -46,6 +46,17 @@ static class PlayerTabUpdateAvailableColorsPatch {
 				return false;
 			}
 		}
+		
+		[HarmonyPatch(typeof(ShipStatus), nameof(ShipStatus.GetSpawnLocation))]
+        	public static class ShipStatusGetSpawnLocationPatch
+		{
+		    public static bool Prefix(ShipStatus __instance, [HarmonyArgument(0)] ref int playerId, [HarmonyArgument(1)] ref int numPlayer, [HarmonyArgument(2)] bool initialSpawn)
```

Comment:
you can leave only required arguments here, harmony will patch it anyway (unless it's overrided)